src/main/scala/strawman/collection/mutable/Seq.scala

```diff
@@ -0,0 +1,91 @@
+package strawman.collection.mutable
+
+import java.lang.IndexOutOfBoundsException
+import scala.{Int, Long, Unit, Boolean, Array}
+import strawman.collection
+import strawman.collection.{IterableOnce, toNewSeq, toOldSeq}
+import scala.Predef.intWrapper
+
+trait Seq[A] extends collection.Seq[A]
+                with collection.SeqLike[A, Seq]
+                with Iterable[A] {
+  def update(idx: Int, elem: A): Unit
+  def mapInPlace(f: A => A): this.type
+}
+
+trait GrowableSeq[A] extends Seq[A]
+                        with collection.SeqLike[A, Seq]
+                        with Growable[A] {
+  def insert(idx: Int, elem: A): Unit
+  def insertAll(idx: Int, elems: IterableOnce[A]): Unit
+  def remove(idx: Int): A
+  def remove(from: Int, n: Int): Unit
+  def flatMapInPlace(f: A => IterableOnce[A]): this.type
+  def filterInPlace(p: A => Boolean): this.type
+  def patchInPlace(from: Int, patch: collection.Seq[A], replaced: Int): this.type
+
+  // +=, ++=, clear inherited from Growable
+  def +=:(elem: A): this.type = { insert(0, elem); this }
+  def +=:(elem1: A, elem2: A, elems: A*): this.type = elem1 +=: elem2 +=: elems.toStrawman ++=: this
```

Comment:
I like symmetry, but I'm not so sure that this isn't confusing.  We deprecated this style of left-assignment-in-place somewhere around 2.10, I think because it felt confusing to people and in practice wasn't really used; it's not in the current collections library any more.  I'd recommend leaving all of the `+=:` variants out.